src/clr-angular/progress/spinner/spinner.ts

```diff
@@ -50,6 +49,10 @@ export class ClrSpinner {
     return 'polite';
   }
 
+  // Get class values before changing it
+  @Input('class') userClasses: string = '';
```

Comment:
Unfortunately, we cannot do this because users could define classes through bindings like this: `[class.user-class]` and `[ngClass]="..."`, which won't be registered here. We shouldn't really try to manipulate classes in bulk here. As I mentioned in the other PR, we need to use HostBinding on each of the properties. That's safe Angular way.